_redirects

```diff
@@ -1,3 +1,4 @@
+/guides/legacy/understanding-state  /docs/deploy/understanding-state 301
```

Comment:
```suggestion
/guides/legacy/understanding-state  /docs/deploy/understand-state 301
```

Reply:
sugg: since we're changing the route anyway, might as well use a verb (vs gerund) for shorter lengths